css/main.css

```diff
@@ -145,68 +149,77 @@ kbd {
 	display: block;
 }
 
+.flexTopBar {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	top: 0;
+	left: 0;
+	z-index: 1;/*Lets topbar render ontop of iframe*/
+	user-select: none;
+}
+
+.spectateBanner::before{
+	content: "👁️ ";
+	font-style: normal;
+}
 .spectateBanner{
 	background-color: #90FF90;
 	cursor: pointer;
-	text-align:center;
+	flex: 0 0 35%;
+	width: 99.5%; /* I think this is due to a padding issue or something? */
+	text-align: center;
+	font-style: italic;
+	background-color: lightgreen;
+	border-style: outset;
+	border-color: lightgreen;
+	font-size: 1em;
+	box-shadow: 0 4px 8px 0 rgba(0,0,0,0.1);
 }
 
 .contentContainer{
 	display: flex;
-	flex-direction: row-reverse;
-	margin-top: var(--topbar-height); /*save room for topbar*/
+	height: 100%;
+}
+
+.sidebarNirnroute{
+	margin-left: 0.5em;
```

Comment:
why is nirnroute different from the rest of the pages?

Reply:
In the guide page, I liked the divider being part of the border of the sidebar so there wasn't a weird gap between the divider and the toolbar part of the sidebar frame. On the nirnroute page though, the lack of a margin makes text directly touch that bar. I suppose an alternate fix is to overhaul the css for that entire nirnroot info panel, but I haven't gotten to that yet so a simple added margin to it fixed the problem.